src/message/html/js.js

```diff
@@ -28,7 +26,21 @@ document.addEventListener('message', function(e) {
       var first = document.getElementById('message-list');
       var before = document.createElement('div');
       first.innerHTML = msg.content;
-      document.body.insertBefore(before, first);
+    case 'fetchingOlder':
+      var element = document.getElementById('spinner-older');
+      if (msg.isVisible) {
+        element.classList.remove('hidden');
```

Comment:
Use the function `toggle` instead of `if add remove`.
https://developer.mozilla.org/en-US/docs/Web/API/Element/classList